src/orca-jedi/increment/Increment.h

```diff
@@ -8,6 +8,7 @@
 #include <string>
 #include <memory>
 #include <vector>
+#include <tuple>
```

Comment:
I think you aren't using tuple any more, but I might have missed it.

Reply:
You're right - removed.